package.json

```diff
@@ -68,13 +72,13 @@
   },
   "scripts": {
     "build:evergreen": "node .evergreen/generate_evergreen_tasks.js",
-    "build:types": "tsc",
+    "build:types": "gulp -f test/tools/ts-pipeline.js",
```

Comment:
Publishing our own types will have to wait until we've completed conversion, so nit: lets rename this to `build:ts`

I can't comment on it but you should delete line 9 (types/index.d.ts)